src/main/java/org/testng/xml/internal/XmlSuiteUtils.java

```diff
@@ -97,6 +127,12 @@ static void ensureNoDuplicateTestsArePresent(XmlSuite xmlSuite) {
             }
         }
     }
+    
+    private static void addNotNullSuite(XmlSuite xmlSuite) {
```

Comment:
Nit picking. Maybe we could name this to `addIfNotNull()`

Reply:
Done